### PR TITLE
Ported the variable payload clusterbangs from sayustation

### PR DESCRIPTION
**Pull request by Menshin**
As suggested in #7120, i've ported (an adapted version of) the variable payload clusterbang from **_sayustation**_.

It's basically a clusterbang, which can spawn any grenade type.
Actually coded in :
- traditional (flashbangs)
- emp
- incendiary
- metal foam
- smoke
- antiweed
- cleaner
- tear gas
- facid
- syndie mini bomb
- Viscerator spawner
- Spesscarp spawner

The only clusterbuster legally accessible in-game is the traditionnal flashbang one, every others are only spawnable by (b)admins.

Needless to say, this has a _huge_ potential for grieving, but we trust the (b)admins to found an event/RP use for these.

### CODE REVIEW

File: code/game/objects/items/weapons/grenades/clusterbuster.dm [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+/obj/item/weapon/grenade/clusterbuster/metalfoam
+	name = "Instant Concrete"
+	payload = /obj/item/weapon/grenade/chem_grenade/metalfoam
+
+/obj/item/weapon/grenade/clusterbuster/inferno
+	name = "Inferno"
+	payload = /obj/item/weapon/grenade/chem_grenade/incendiary
+
+/obj/item/weapon/grenade/clusterbuster/antiweed
+	name = "RoundDown"
```
Comment (Iamgoofball) [inline code/game/objects/items/weapons/grenades/clusterbuster.dm]: spacesanto will sue you

**Comment by Iamgoofball** [comment]
:+1: merge yesterday

**Comment by CollenN** [comment]
This is fine with me [+1]

**Comment by Dorsisdwarf** [comment]
>Cluster minibombs

All of the yes.

**Comment by Profakos** [comment]
But what about the most important one, the parrot grenade... (it is not important)

**Comment by Menshin** [comment]
It's planned, this first version (limited to grenades) to gather feedback and some live testing (if it ever get merged).

File: code/game/objects/items/weapons/grenades/clusterbuster.dm [hunk 2]
```diff
@@ -0,0 +1,123 @@
+////////////////////
+//Clusterbang
+////////////////////
+/obj/item/weapon/grenade/clusterbuster
+	desc = "Use of this weapon may constiute a war crime in your area, consult your local captain."
```
Comment (Cheridan) [inline code/game/objects/items/weapons/grenades/clusterbuster.dm]: constiute :^)

Comment (Tenebrosity) [inline code/game/objects/items/weapons/grenades/clusterbuster.dm]: >wasn't fix

Fug :DDD

**Pull request merged**